<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryLimit</class>
 <widget class="QWidget" name="InventoryLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
{
background-color: rgb(180, 85, 100);
color:white;
}

QPushButton:hover:pressed
{
background-color:rgb(180, 85, 100);
color:white;
}

QPushButton
{
background-color:rgb(84, 115, 135);
color:white;
font-size:12px;
width: 150px;
border-radius:2px;
spacing:2px;
padding: 0px 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widgetControl" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <widget class="QPushButton" name="pushButtonNoSelect">
        <property name="geometry">
         <rect>
          <x>171</x>
          <y>31</y>
          <width>150</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 全不选</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/image/image/全不选.png</normaloff>:/image/image/全不选.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSetUp">
        <property name="geometry">
         <rect>
          <x>327</x>
          <y>31</y>
          <width>150</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 批量设置供应商</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/image/image/确认.png</normaloff>:/image/image/确认.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="labelSize">
        <property name="geometry">
         <rect>
          <x>653</x>
          <y>11</y>
          <width>16</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAllSelect">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>31</y>
          <width>150</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string> 全选</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/image/image/批量.png</normaloff>:/image/image/批量.png</iconset>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>原料名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>条形码</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>库存单位</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上限</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>下限</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
